{{cookiecutter.project_slug}}/cookiecutter-config-file.yml

```diff
@@ -1,5 +1,6 @@
 default_context:
   project_name: '{{ cookiecutter.project_name }}'
+  project_version: '{{ cookiecutter.project_version }}'
```

Comment:
```suggestion
```

Please remove the project version, that would be a default of `0.1.0`.

Reply:
Project version need to poetry pyproject.toml. I can fix it in this file. It's ok?